### PR TITLE
Aura Arabic localization

### PR DESCRIPTION
**Pull request by array-in-a-matrix**
Hello, I have translated Aura to Arabic!
I haven't compiled the code to check if the translation shows up correctly.
There might be a problem when displaying Arabic text that also include English letters however I haven't verified that.
I hope my contribution helps!

### CODE REVIEW

File: aura/lib/Aura/Languages.hs [hunk 1]
```diff
@@ -917,14 +967,16 @@ cleanStates_5 t = \case
 cleanStates_6 :: Int -> Language -> Doc AnsiStyle
 cleanStates_6 n = \case
   Polish    -> pretty n <+> "jest przypiętych i nie zostanie usuniętych."
+  Arabic    -> ".اذا كانو مثبتين ولا يمكن ازالتهم " <+> pretty n
   Spanish   -> pretty n <+> "de estos están anclados y no se eliminarán."
   Italian   -> pretty n <+> "di questi sono stati fissati, perciò non saranno rimossi."
   Dutch     -> pretty n <+> "hiervan zijn vastgezet, en worden niet verwijderd."
-  _         -> pretty n <+> "of these are pinned, and won't be removed."
```
Comment (micwoj92) [inline aura/lib/Aura/Languages.hs:923]: why did you change that?

Comment (fosskers) [reply aura/lib/Aura/Languages.hs:923]: It's a reference to this issue https://github.com/fosskers/aura/issues/725

But the original wording was correct and intended. Please do revert this change.

**Review by fosskers — CHANGES_REQUESTED** [review]
Thank you very much for this! I've been waiting a long time for somebody to finally add Arabic :)

You could also add your name to the list of translators in `Languages.hs`, the README, and the man page. Otherwise I can add it for you.

**Review by array-in-a-matrix — COMMENTED** [review]
My apologies, I have corrected that.

**Comment by fosskers** [comment]
Thank you! I will merge this soon and make a new release.

**Comment by array-in-a-matrix** [comment]
No problem! I have always wanted to contribute to the FOSS community. 
I have credited myself in the README and in `Languages.hs` however I don't know where the man page is.

**Comment by fosskers** [comment]
It's at `aura/doc/aura.8`.

**Comment by array-in-a-matrix** [comment]
It's done.

**Comment by fosskers** [comment]
Thanks again!
